akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala

```diff
@@ -99,6 +99,4 @@ import akka.stream.scaladsl.Source
     composedSource
   }
 
-  // FIXME
-  def processElement(elt: StreamElement)(implicit ec: ExecutionContext): Future[Done] = ???
```

Comment:
removed this because it was in the way and will be replaced by the approach in #53